scripts/create-release-branch.js

```diff
@@ -102,5 +92,21 @@ shell.exec('git add lerna.json');
 // Git commit
 shell.exec(`git commit -m "chore: update to version ${nextVersion}"`);
 
+// Merge release branch to develop
+shell.exec(`git checkout develop`)
+shell.exec(`git merge ${releaseBranchName}`)
+shell.exec(`git push -u origin develop`)
+shell.exec(`git checkout ${releaseBranchName}`)
+
+// Generate changelog
+const previousBranchName = changeLogGeneratorUtils.getPreviousReleaseBranch(releaseBranchName);
+const parsedCommits = changeLogGeneratorUtils.parseCommits(changeLogGeneratorUtils.getCommits(releaseBranchName, previousBranchName));
+const groupedMessages = changeLogGeneratorUtils.getMessagesGroupedByPackage(parsedCommits, '');
+const changeLog = changeLogGeneratorUtils.getChangeLogText(releaseBranchName, groupedMessages);
+changeLogGeneratorUtils.writeChangeLog(changeLog);
+
+const commitCommand = `git commit -a -m "chore: generated CHANGELOG for ${releaseBranchName}"`;
+shell.exec(commitCommand);
```

Comment:
Do we want to merge into develop before we commit the change log?  I fee like that kinda was the point, but hurts my head a little :)

Reply:
I went back and looked at the ticket and it does indeed say to do exactly what you're doing. I do feel like we need a code comment here to call out why we do that b/c if I were to revisit this code in 6 months I'll def be tempted to change the order.

Reply:
@gbockus-sf  Added some comment here.

Reply:
@gbockus-sf  Added some comment here.